### PR TITLE
changed info message for when callback url is present

### PR DESCRIPTION
**Pull request by Raysonviswas**
check first commit for changes

### CODE REVIEW

File: app/uk/gov/hmrc/customs/notification/controllers/HeaderValidator.scala [hunk 1]
```diff
@@ -17,7 +17,7 @@
 package uk.gov.hmrc.customs.notification.controllers
 
 import play.api.http.HeaderNames._
-import play.api.mvc.{ActionBuilder, AnyContent, BodyParser, ControllerComponents, Headers, Request, Result}
+import play.api.mvc._
```
Comment (paulblakehmrc) [inline app/uk/gov/hmrc/customs/notification/controllers/HeaderValidator.scala]: Could you revert this import change and the others in this PR? It just introduces noise in the commit. Also, explicit imports are better than wildcard.

File: app/uk/gov/hmrc/customs/notification/services/CustomsNotificationMetricsService.scala [hunk 2]
```diff
@@ -22,8 +22,8 @@ import uk.gov.hmrc.customs.notification.domain.{CustomsNotificationsMetricsReque
 import uk.gov.hmrc.customs.notification.logging.NotificationLogger
 import uk.gov.hmrc.customs.notification.util.DateTimeHelpers._
 
-import scala.util.control.NonFatal
 import scala.concurrent.{ExecutionContext, Future}
+import scala.util.control.NonFatal
```
Comment (paulblakehmrc) [inline app/uk/gov/hmrc/customs/notification/services/CustomsNotificationMetricsService.scala]: revert import change

File: app/uk/gov/hmrc/customs/notification/services/PushOrPullService.scala [hunk 3]
```diff
@@ -19,7 +19,7 @@ package uk.gov.hmrc.customs.notification.services
 import javax.inject.Inject
 import reactivemongo.bson.BSONObjectID
 import uk.gov.hmrc.customs.notification.connectors.{ApiSubscriptionFieldsConnector, MapResultError, NotificationQueueConnector}
-import uk.gov.hmrc.customs.notification.domain.{ApiSubscriptionFields, ClientId, ClientNotification, ClientSubscriptionId, DeclarantCallbackData, HasId, NonHttpError, NotificationWorkItem, PushNotificationRequest, PushNotificationRequestBody, ResultError}
+import uk.gov.hmrc.customs.notification.domain._
```
Comment (paulblakehmrc) [inline app/uk/gov/hmrc/customs/notification/services/PushOrPullService.scala]: revert import change

File: test/unit/controllers/HeaderValidatorSpec.scala [hunk 4]
```diff
@@ -16,8 +16,8 @@
 
 package unit.controllers
 
-import org.scalatestplus.mockito.MockitoSugar
 import org.scalatest.prop.TableDrivenPropertyChecks
+import org.scalatestplus.mockito.MockitoSugar
```
Comment (paulblakehmrc) [inline test/unit/controllers/HeaderValidatorSpec.scala]: revert import change

File: test/unit/services/WorkItemProcessingSchedulerSpec.scala [hunk 5]
```diff
@@ -20,8 +20,8 @@ import akka.actor.ActorSystem
 import org.mockito.Mockito._
 import org.scalatest.concurrent.Eventually
 import org.scalatest.concurrent.PatienceConfiguration.{Interval, Timeout}
-import org.scalatestplus.mockito.MockitoSugar
 import org.scalatest.time.{Millis, Seconds, Span}
+import org.scalatestplus.mockito.MockitoSugar
```
Comment (paulblakehmrc) [inline test/unit/services/WorkItemProcessingSchedulerSpec.scala]: revert import change

File: test/unit/controllers/HeaderValidatorSpec.scala (hunk 4 above)
Comment (Raysonviswas) [reply test/unit/controllers/HeaderValidatorSpec.scala]: needs mockitoSugar to mock notificatiuon logger

File: test/unit/services/WorkItemProcessingSchedulerSpec.scala (hunk 5 above)
Comment (Raysonviswas) [reply test/unit/services/WorkItemProcessingSchedulerSpec.scala]: needs mockitoSugar to mock notificatiuon logger

File: app/uk/gov/hmrc/customs/notification/services/CustomsNotificationMetricsService.scala (hunk 2 above)
Comment (Raysonviswas) [reply app/uk/gov/hmrc/customs/notification/services/CustomsNotificationMetricsService.scala]: needed for pull function

**Pull request merged**